rstdocs/index.rst

```diff
@@ -174,6 +174,21 @@ using sets as parameters.
 Nested structures are also not directly allowed but you can use `json <https://www.edgedb.com/docs/edgeql/insert#bulk-inserts>`_
 instead.
 
+By default EdgeDB will ignore embedded structs when marshaling/unmarshaling.
+To treat an embedded struct's fields as part of the parent struct's fields
```

Comment:
Make sure I've interpreted this correctly. I wasn't 100% certain how to interpret it.
```suggestion
To treat an embedded struct's fields as part of the parent struct's fields,
```